webpack.config.js

```diff
@@ -7,7 +7,7 @@ const config = require('./config/webpack.common.js');
 webpackConfig.devServer = {
     contentBase: config.paths.public,
     hot: true,
-    port: 8002,
+    port: 8003,
```

Comment:
Why is this being updated?

Reply:
Yeah my bad, I had o change the port so I can have multiple apps running at the same time.